assets/js/global.js

```diff
@@ -40,28 +40,27 @@ jQuery( document ).ready( function( $ ) {
 	}
 
 	function adjustHeaderHeight() {
-
-		$navigationHeight = $navigation.outerHeight();
-
 		if ( 'none' === $( '.menu-toggle' ).css( 'display' ) ) {
 			$branding.css( 'margin-bottom', $navigationHeight );
 		} else {
 			$branding.css( 'margin-bottom', '0' );
 		}
 	}
 
+	function getNavProps() {
```

Comment:
I would suggest changing this function name. A 'get' in the function name implies the function will return something. Maybe `setNavProps()`?